spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -348,3 +357,18 @@ trait CommonJobExecution extends ProjectMetadata {
     }
   }
 }
+
+object CommonJobExecution {
+
+  /**
+   * will create java opts string e.g. "-Dkey.one=value.one -Dkey.two=value.two"
+   *
+   * @param configMap
+   * @return
+   */
+  def javaOptsStringFromConfigMap(configMap: Map[String, String]): String = {
```

Comment:
I would suggest to have this object or function somewhere else, since the file is quite large already

Reply:
Possibly. `SecureConfig` (is and could be used for data from it)? `ConfigUtils` (because it can be general)?